data/serializer/src/main/java/tech/pegasys/teku/api/response/v1/teku/GetPreparedBeaconProposersResponse.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2022 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.api.response.v1.teku;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+import java.util.Map;
+
+public class GetPreparedBeaconProposersResponse {
+  private final List<Map<String, Object>> data;
+
+  @JsonCreator
+  public GetPreparedBeaconProposersResponse(
+      @JsonProperty("data") final List<Map<String, Object>> data) {
+    this.data = data;
+  }
```

Comment:
this is going to mean we don't have any swagger context for the type of data returned... It would basically end up with noone being able to generate an api to use with this endpoint, so it may actually stop people trying it...

Are there a lot of different types that would need defining, or would it just be a new schema object or 3 I guess is the question...

Reply:
yeah, the reason I simplified like that is because there is a bit of pain due to circular dependencies. I'll spend some more time to find a way to overcome them.

Reply:
just create the module internally there if it's easier... should be ok...